src/app/tests/suites/certification/Test_TC_S_3_1.yaml

```
@@ -0,0 +1,236 @@
+
+          [1702307872.231804][4852:4854] CHIP:DMG: Received Command Response Data, Endpoint=EndpointID Cluster=0x0000_0005 Command=0x0000_0000
+          [1702307872.231895][4852:4854] CHIP:TOO: Endpoint: EndpointID Cluster: 0x0000_0005 Command 0x0000_0000
+          [1702307872.232137][4852:4854] CHIP:TOO:   AddSceneResponse: {
+          [1702307872.232800][4852:4854] CHIP:TOO:     status: 0
+          [1702307872.232873][4852:4854] CHIP:TOO:     groupID: GroupID
+          [1702307872.232928][4852:4854] CHIP:TOO:     sceneID: SceneID1
+          [1702307872.232976][4852:4854] CHIP:TOO:    }
+      disabled: true
+
+    - label: "Step 2: DUT issues an C_VIEW_SCENE command to the Test Harness."
+      PICS: S.C.C01.Tx
+      verification: |
+          DUT issues an C_VIEW_SCENE command to the Test Harness equivalent to the following chip-tool command:
+          ./chip-tool scenes view-scene GroupID SceneID1 NodeID EndpointID
+
+          Verify the DUT receives the view-scenes response and that the extension fields in the log match the ones expected. The number of ExtensionFieldSets, the value of clusterId of each ExtensionFieldSet,
+          The number of ExtensionFieldSets, the value of clusterId of each ExtensionFieldSet, the number of attributes in attributeValueList and their values varies for each application.
+
+          [1702308535.141808][4874:4876] CHIP:DMG: Received Command Response Data, Endpoint=EndpointID Cluster=0x0000_0005 Command=0x0000_0001
+          [1702308535.141865][4874:4876] CHIP:TOO: Endpoint: EndpointID Cluster: 0x0000_0005 Command 0x0000_0001
+          [1702308535.141999][4874:4876] CHIP:TOO:   ViewSceneResponse: {
+          [1702308535.142032][4874:4876] CHIP:TOO:     status: 0
+          [1702308535.142058][4874:4876] CHIP:TOO:     groupID: GroupID
+          [1702308535.142084][4874:4876] CHIP:TOO:     sceneID: SceneID1
+          [1702308535.142110][4874:4876] CHIP:TOO:     transitionTime: TransitionTime
+          [1702308535.142138][4874:4876] CHIP:TOO:     sceneName: Scene Name
+          [1702308535.142176][4874:4876] CHIP:TOO:     extensionFieldSets: x entries
+          [1702308535.142220][4874:4876] CHIP:TOO:       [ClusterCount]: {
+          [1702308535.142248][4874:4876] CHIP:TOO:         ClusterID: clusterId
+          [1702308535.142279][4874:4876] CHIP:TOO:         AttributeValueList: x entries
+          [1702308535.143760][4874:4876] CHIP:TOO:           [clusterX]: {
+          [1702308535.143796][4874:4876] CHIP:TOO:             AttributeID: attributeId
+          [1702308535.143825][4874:4876] CHIP:TOO:             AttributeValue: attributeValue
+          [1702308535.143853][4874:4876] CHIP:TOO:            } ...
+          [1702308535.143881][4874:4876] CHIP:TOO:        }
+          [1702308535.143910][4874:4876] CHIP:TOO:    }
+      disabled: true
+
+    - label: "Step 3: DUT issues an C_REMOVE_SCENE command to the Test Harness."
```

```suggestion
    - label: "Step 3: DUT issues a C_REMOVE_SCENE command to the Test Harness."
```
this applies to all the other locations